nomic/atlas.py

```diff
@@ -271,6 +327,14 @@ def add_embeddings(
             raise ValueError(msg)
 
         progressive = len(project['atlas_indices']) > 0
+        try:
+            self._validate_user_supplied_metadata(data=data, project=project)
+        except BaseException as e:
+            if not progressive:
+                self.delete_project(project_id=project_id)
```

Comment:
not sure we want to do this here. may be confusing if a user is not going through the map_embeddings interface